resources/benchmark-runner.mjs

```diff
@@ -147,57 +172,54 @@ export class BenchmarkRunner {
         }, 0);
     }
 
-    step(state) {
+    async step(state) {
```

Comment:
Can we do this `async` syntax adoption in a separate patch?
It's making the code review harder.